src/App.js

```diff
@@ -381,6 +382,8 @@ export class App extends React.Component<AppProps, AppState> {
             isDraggingCommand: false,
             audioEnabled: true,
             announcementsEnabled: true,
+            // Keep utteranceRate in the state, which later have a controller to change
```

Comment:
"so that we can add controls to change it later"?

Reply:
that sounds way more clear!